### ISSUE TITLE
Changing it to "Bangla", BN, bn / Bangla, Bengali

### ISSUE THREAD

**Issue by raqueeb** [squidfunk/mkdocs-material#1774]
## Instructions

1. Check, if your language is already available: [here](https://bit.ly/3c2ox0C)
2. If it isn't, please translate the labels on the right:

``` jinja
{% macro t(key) %}{{ {
  "language": "bd",
  "direction": "ltr",
  "clipboard.copy": "ক্লিপবোর্ডে কপি করুন",
  "clipboard.copied": "ক্লিপবোর্ডে কপি হয়েছে",
  "edit.link.title": "এই পেজ এডিট করুন",
  "footer.previous": "পূর্ববর্তী",
  "footer.next": "পরে",
  "footer.title": "ফুটার",
  "header.title": "হেডার",
  "meta.comments": "কমেন্ট",
  "meta.source": "সোর্স",
  "nav.title": "ন্যাভিগেশন",
  "search.config.lang": "bd",
  "search.config.pipeline": "trimmer, stopWordFilter",
  "search.config.separator": "[\s\-]+",
  "search.placeholder": "সার্চ",
  "search.reset": "মুছে ফেলুন",
  "search.result.placeholder": "সার্চ টাইপ করুন",
  "search.result.none": "কিছু পাওয়া যায়নি",
  "search.result.one": "১ টা ডকুমেন্ট",
  "search.result.other": "# টা ডকুমেন্ট",
  "skip.link.title": "কনটেন্টে যান",
  "source.link.title": "রিপোজিটরিতে যান",
  "source.revision.date": "শেষ আপডেট",
  "tabs.title": "ট্যাব",
  "toc.title": "টেবিল অফ কনটেন্ট"
}[key] }}{% endmacro %}
```

Thanks!

**Comment by squidfunk** [comment]
Thanks! I'll integrate it in the next release 😊

**Comment by squidfunk** [comment]
Added in 3f4e5615

**Comment by squidfunk** [comment]
Released as part of 5.3.3

**Comment by raqueeb** [comment]
Thank you for your support.
